lectures/markov_chains_II.md

```diff
@@ -295,6 +295,7 @@ The transition graph shows that this model is irreducible.
 ```{image} /_static/lecture_specific/markov_chains_II/example4.png
 :name: mc_example4
 :align: center
+:scale: 80%
```

Comment:
Many thanks @mmcky,

The code looks great to me. 

On my side, the image looks slightly too wide:

<img width="804" alt="Screenshot 57" src="https://github.com/QuantEcon/lecture-python-intro/assets/39026988/3aa28fbb-8312-492b-8512-29692677ff5b">

Should we adjust the aspect ratio to make it narrower?

Reply:
I think https://github.com/QuantEcon/lecture-python-intro/issues/394#issuecomment-2058145341 should fix the error. It would also keep it consistent with other figures in the lecture.